app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -79,6 +86,60 @@ class RevisionCardFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  private fun setUpRevisionNavigationCards(
+    binding: RevisionCardFragmentBinding,
+    topicId: String,
+    subtopicId: Int,
+    subtopicListSize: Int
+  ) {
+
+    binding.prevSubtopicImageView.setEntityType(entityType)
+    binding.nextSubtopicImageView.setEntityType(entityType)
+
+    if (subtopicId == 1) {
+      binding.previousNavigationCard.visibility = View.INVISIBLE
```

Comment:
Suggest instead making this a bit more explicit, such as:

```kotlin
binding.previousNavigationCard.visibility = if (subtopicId == 1) View.INVISIBLE else View.VISIBLE
```

This makes it specifically clear when the card is visible vs. not visible.

That being said, why are we using INVISIBLE instead of GONE? If these will never be visible in this situation, it doesn't seem to make sense to pay the extra measuring/layout cost since the view shouldn't be showing up.

Ditto for the next card below.

Reply:
done